classes/02class/exercises/c02-network04/raghunadhpokkalath/README.MD

```diff
@@ -0,0 +1,318 @@
+    }
+}
+> aws ec2 create-tags --resources subnet-0e22678e3713193d2 --tags Key=Name,Value=private-subnet-2c
+
+> aws ec2 describe-subnets --filters "Name=subnet-id,Values=subnet-0e22678e3713193d2"
+{
+    "Subnets": [
+        {
+            "AvailabilityZone": "ap-southeast-2c",
+            "AvailabilityZoneId": "apse2-az2",
+            "AvailableIpAddressCount": 251,
+            "CidrBlock": "10.0.3.0/24",
+            "DefaultForAz": false,
+            "MapPublicIpOnLaunch": false,
+            "State": "available",
+            "SubnetId": "subnet-0e22678e3713193d2",
+            "VpcId": "vpc-076a05ec0f3be4895",
+            "OwnerId": "361529347535",
+            "AssignIpv6AddressOnCreation": false,
+            "Ipv6CidrBlockAssociationSet": [],
+            "Tags": [
+                {
+                    "Key": "Name",
+                    "Value": "private-subnet-2c"
+                }
+            ],
+            "SubnetArn": "arn:aws:ec2:ap-southeast-2:361529347535:subnet/subnet-0e22678e3713193d2"
+        }
+    ]
+}
+
+Public subnet [CIDR - 10.0.210.0/24]     
+
+> aws ec2 create-subnet --vpc-id vpc-076a05ec0f3be4895  --availability-zone ap-southeast-2c --cidr-block 10.0.210.0/24
+{
+    "Subnet": {
+        "AvailabilityZone": "ap-southeast-2c",
+        "AvailabilityZoneId": "apse2-az2",
+        "AvailableIpAddressCount": 251,
+        "CidrBlock": "10.0.210.0/24",
+        "DefaultForAz": false,
+        "MapPublicIpOnLaunch": false,
+        "State": "available",
+        "SubnetId": "subnet-0b2095e1b5117df04",
+        "VpcId": "vpc-076a05ec0f3be4895",
+        "OwnerId": "361529347535",
+        "AssignIpv6AddressOnCreation": false,
+        "Ipv6CidrBlockAssociationSet": [],
+        "SubnetArn": "arn:aws:ec2:ap-southeast-2:361529347535:subnet/subnet-0b2095e1b5117df04"
+    }
+}
+> aws ec2 create-tags --resources subnet-0b2095e1b5117df04 --tags Key=Name,Value=public-subnet-2c
+
+> aws ec2 describe-subnets --filters "Name=subnet-id,Values=subnet-0b2095e1b5117df04"
+
+{
+    "Subnets": [
+        {
+            "AvailabilityZone": "ap-southeast-2c",
+            "AvailabilityZoneId": "apse2-az2",
+            "AvailableIpAddressCount": 251,
+            "CidrBlock": "10.0.210.0/24",
+            "DefaultForAz": false,
+            "MapPublicIpOnLaunch": false,
+            "State": "available",
+            "SubnetId": "subnet-0b2095e1b5117df04",
+            "VpcId": "vpc-076a05ec0f3be4895",
+            "OwnerId": "361529347535",
+            "AssignIpv6AddressOnCreation": false,
+            "Ipv6CidrBlockAssociationSet": [],
+            "Tags": [
+                {
+                    "Key": "Name",
+                    "Value": "public-subnet-2c"
+                }
+            ],
+            "SubnetArn": "arn:aws:ec2:ap-southeast-2:361529347535:subnet/subnet-0b2095e1b5117df04"
+        }
+    ]
+}
+```
+
+- Any extra challenges faced?
+```
+Finding out the command to filter subnet after creation
```

Comment:
Nice work doing that. you can also add `--query Subnets[*].SubnetId` if you wanted to see only the ID of the subnets. Just an example :-)